### PR TITLE
fix typos in examples

### PR DESCRIPTION
**Pull request by smsaladi**

### CODE REVIEW

File: mdtraj/formats/dcd/dcd.pyx [hunk 1]
```diff
@@ -170,15 +170,15 @@ cdef class DCDTrajectoryFile:
     >>> f.close()
 
     >>> # read all of the data with automatic closing of the file
-    >>> with DCDTrajectoryFile('mytrajectory.dcd') as f:
```
Comment (smsaladi) [inline mdtraj/formats/dcd/dcd.pyx]: Actually, I'm not sure if this line was purposefully written like this or not...

File: mdtraj/formats/dcd/dcd.pyx [hunk 2]
```diff
@@ -170,15 +170,15 @@ cdef class DCDTrajectoryFile:
...
+    >>> with DCDTrajectoryFile('mytrajectory.dcd', 'r') as f:
```
Comment (smsaladi) [inline mdtraj/formats/dcd/dcd.pyx]: (Proposed change)

Comment (mpharrigan) [inline mdtraj/formats/dcd/dcd.pyx]: Right, since the 'r' is default. Don't think it matters that much

**Comment by mpharrigan** [comment]
Thanks!

**Pull request merged**